scripts/modal.js

```diff
@@ -1,3 +1,133 @@
+let htmlElements = '';
+
+const projects = [
+  {
+    id: 'projects-wrapper',
+    name: 'Project name goes here',
+    demo: 'https://ksigei.github.io/portfolio2/',
+    source: 'https://github.com/ksigei/portfolio2',
+    langs: {
+      html: 'HTML/CSS',
+      ruby: 'Ruby on Rails',
+      js: 'JavaScript',
+    },
+  },
+  {
+    id: 'projects-wrapper',
+    name: 'Project name goes here',
+    demo: 'https://ksigei.github.io/portfolio2/',
+    source: 'https://github.com/ksigei/portfolio2',
+    langs: {
+      html: 'HTML/CSS',
+      ruby: 'Ruby on Rails',
+      js: 'JavaScript',
+    },
+  },
+  {
+    id: 'projects-wrapper',
+    name: 'Project name goes here',
+    demo: 'https://ksigei.github.io/portfolio2/',
+    source: 'https://github.com/ksigei/portfolio2',
+    langs: {
+      html: 'HTML/CSS',
+      ruby: 'Ruby on Rails',
+      js: 'JavaScript',
+    },
+  },
+  {
+    id: 'projects-wrapper',
+    name: 'Project name goes here',
+    demo: 'https://ksigei.github.io/portfolio2/',
+    source: 'https://github.com/ksigei/portfolio2',
+    langs: {
+      html: 'HTML/CSS',
+      ruby: 'Ruby on Rails',
+      js: 'JavaScript',
+    },
+  },
+  {
+    id: 'projects-wrapper',
+    name: 'Project name goes here',
+    demo: 'https://ksigei.github.io/portfolio2/',
+    source: 'https://github.com/ksigei/portfolio2',
+    langs: {
+      html: 'HTML/CSS',
+      ruby: 'Ruby on Rails',
+      js: 'JavaScript',
+    },
+  },
+  {
+    id: 'projects-wrapper',
+    name: 'Project name goes here',
+    demo: 'https://ksigei.github.io/portfolio2/',
+    source: 'https://github.com/ksigei/portfolio2',
+    langs: {
+      html: 'HTML/CSS',
+      ruby: 'Ruby on Rails',
+      js: 'JavaScript',
+    },
+  },
+  {
+    id: 'projects-wrapper',
+    name: 'Project name goes here',
+    demo: 'https://ksigei.github.io/portfolio2/',
+    source: 'https://github.com/ksigei/portfolio2',
+    langs: {
+      html: 'HTML/CSS',
+      ruby: 'Ruby on Rails',
+      js: 'JavaScript',
+    },
+  },
+];
+
+for (let i = 0; i <= 6; i += 1) {
+  htmlElements
+   += ` 
+   <div class="modal" id="modal2" data-animation="mixInAnimations">
```

Comment:
- [ ] Kindly add the right background color and remove horizontal scrolls from your website, Horizontal scrolling especially for pages with lots of text or buttons to click can create a bad  user experience for users if users have to scroll horizontally to see the remaining content of the page or to click the button so its best to avoid it in your website, Kindly fix this